### PR TITLE
Acc 1610 node upgrade

### PR DESCRIPTION
**Pull request by danieleruiz**
### Description
Migrate to Node 16, unless it’s a lambda in which case you should migrate to Node 14. Follow this migration guide: https://apaleslimghost.notion.site/Node-14-16-Migration-Guide-7d69bb88065c4afa95b51b0c8052ade2

### Ticket
[TICKET](https://financialtimes.atlassian.net/browse/ACC-1610)
### What is the new version number in package.js?
0.41.1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,7 +53,8 @@
     "proxyquire": "^1.7.4",
     "sinon": "^6.0.0",
     "sinon-chai": "^3.2.0",
-    "snyk": "^1.167.0"
+    "snyk": "^1.167.0",
+    "unzipper": "^0.10.11"
```
Comment (asugar13) [inline package.json:57]: nice. can we remove the other dependency? compress?

File: test/server/lib/download/video.spec.js [hunk 2]
```diff
@@ -438,31 +434,19 @@ describe(MODULE_ID, function () {
...
-			});
-
-			it('captions', function() {
-				expect(captions.data.toString('utf8')).to.equal(dl.captionFiles[0].file);
+				await dl.appendAll();
 			});
 
-			it('media', function() {
-				expect(media.data.equals(fs.readFileSync(path.resolve(`${FIXTURES_DIRECTORY}/video-small.mp4`)))).to.be.true;
+			xit('media', function() {
```
Comment (asugar13) [inline test/server/lib/download/video.spec.js:442]: **Question**: What's the benefit of using xit? Do you have a link to documentation?

Comment (danieleruiz) [reply test/server/lib/download/video.spec.js]: xit is for skip the test , i forgot to remove it. it was for development. thanks!

**Review by asugar13 — APPROVED** [review]
looks good!

**Pull request merged**